Library/Formula/ibex.rb

```diff
@@ -0,0 +1,41 @@
+class Ibex < Formula
+  desc "C++ library for constraint processing over real numbers."
+  homepage "http://www.ibex-lib.org/"
+  url "http://www.ibex-lib.org/sites/default/files/ibex-2.1.16.tar.gz"
+  sha256 "d92ff32f14d27ad7b390ae693beb311b58cf6babccac85bbdaa5f5d0b8648845"
+
+  option "with-java", "Build Java bindings for Choco solver."
+  depends_on :java => ["1.8+", :optional]
+  depends_on "bison" => :build
+  depends_on "flex" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    if build.with? "java"
+      system "./waf", "configure",
+        "--with-jni",
+        "--prefix=#{prefix}"
+    else
+      system "./waf", "configure",
+        "--enable-shared",
+        "--prefix=#{prefix}"
+    end
+    system "./waf", "install"
+
+    # copy and compile examples and benchmark for the test
+    share.install "examples"
```

Comment:
`pkgshare` is new. Where was the error being thrown from, runtime or during the test?

`pkgshare` = prefix/share/formula_name/
`share` = prefix/share

We've started placing examples and similar things in `pkgshare` because there's a risk for accidental conflict installing every formula's examples into the same common prefix - One formula we handled the other day installed an example file to generate just called `example.c`. Installing them into `pkgshare` ensures that conflict can never happen.